app/services/idv/profile_step.rb

```diff
@@ -70,7 +61,6 @@ def extra_analytics_attributes
       {
         idv_attempts_exceeded: throttled?,
         vendor: idv_result.except(:errors, :success),
-        ssn_is_unique: ssn_is_unique?,
```

Comment:
what if we kept this around just for analytics? not enforcing it but still reading it? Might be a useful signal?

We could add a non-unique index on SSN fingerprint so that we can quickly find duplicates across users?